buildkite/bazelci.py

```diff
@@ -2735,13 +2735,9 @@ def create_step(
         ]
     }
 
-    # Retry on macos_arm64 due to
-    # https://github.com/bazelbuild/continuous-integration/issues/2025
-    if platform == "macos_arm64":
```

Comment:
this is no longer important?

Reply:
Yes, that particular issue is no longer happening.